tests/test_sbom.py

```diff
@@ -14,11 +14,11 @@
 @pytest.mark.parametrize(
     ["value", "expected"],
     [
-        ("abc", "abc-ba7816bf"),
```

Comment:
As far as I can tell this doesn't actually exercise the unhappy hash path, would it make sense to add a collision to this set of parameters? I don't remember if pytest guarantees ordered execution of the test with parameters.